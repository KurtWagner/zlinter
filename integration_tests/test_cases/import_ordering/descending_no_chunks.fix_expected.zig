const banana = @import("banana");
const zebra = @import("zebra");
const apple = @import("apple");

// import_order checks per scope, so although this chunk has the wrong order
// it won't be detected until the issues in the above chunk are fixed first.
const cat = @import("cat");

pub fn main() void {
    const banana_main = @import("apple_main");
    const apple_main = @import("apple_main");

    _ = banana_main;
    _ = apple_main;
}
