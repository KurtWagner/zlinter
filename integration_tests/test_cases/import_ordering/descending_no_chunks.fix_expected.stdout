2 fixes applied to: .zig-cache/tmp/import_ordering.input.zig
Fixed 2 issues in 1 files!
0 issues disabled by comments.
